packages/components/molecules/f-navigation-links/src/components/NavigationLinks.vue

```diff
@@ -1,32 +1,78 @@
 <template>
-    <div
+    <ul
+        v-if="links.length"
         :class="$style['c-navigationLinks']"
         data-test-id="navigationLinks">
-        I am a NavigationLinks Component (GB)
-    </div>
+        <li
+            v-for="({ id, url, name, selected }, i) in links"
+            :key="i"
+            :class="$style['c-navigationLink-item']">
+            <span
+                v-if="selected"
+                :data-test-id="id"
+                tabindex="0"
+                :class="[$style['c-navigationLink-link'], $style['c-navigationLink--active']]">
+                {{ name }}
+            </span>
+            <v-link
+                v-else
+                :data-test-id="id"
+                :has-text-decoration="false"
+                :href="url"
+                :class="$style['c-navigationLink-link']">
+                {{ name }}
+            </v-link>
+        </li>
+    </ul>
 </template>
 
 <script>
+import VLink from '@justeat/f-link';
+import '@justeat/f-link/dist/f-link.css';
 
 export default {
     name: 'NavigationLinks',
-    components: {},
+
+    components: {
+        VLink
+    },
+
     props: {
+        links: {
+            type: Array,
+            default: () => []
+        }
     }
 };
 </script>
 
 <style lang="scss" module>
-
 .c-navigationLinks {
-    display: flex;
-    justify-content: center;
-    min-height: 80vh;
-    width: 80vw;
-    margin: auto;
-    border: 1px solid $color-red;
-    font-family: $font-family-base;
-    @include font-size(heading-m);
-}
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+
+    li {
+        display: inline-block;
+        width: 100%;
 
+        .c-navigationLink-link {
+            display: block;
+            padding: spacing() 0 spacing() spacing(x2);
+            border-left: 2px solid $color-grey-30;
+            color: $color-grey;
+
+            &:focus,
+            &:hover {
+                border-left: 2px solid $color-orange;
+                text-decoration: none;
+            }
+        }
+
+        .c-navigationLink--active {
+            border-left: 2px solid $color-orange;
+            font-weight: $font-weight-bold;
+        }
+    }
+}
```

Comment:
Better not to nest these as it will just make the styles more specific. So you could just do:
```suggestion
}

.c-navigationLinks-item {
    display: inline-block;
    width: 100%;
}

.c-navigationLinks-link {
    display: block;
    padding: spacing() 0 spacing() spacing(x2);
    border-left: 2px solid $color-grey-30;
    color: $color-grey;
 
    &:focus,
    &:hover {
        border-left: 2px solid $color-orange;
        text-decoration: none;
    }
 }

.c-navigationLinks-link--active {
        border-left: 2px solid $color-orange;
        font-weight: $font-weight-bold;
    }
}

```